File: Chat-Common/src/main/java/com/summit/chat/model/entity/SysNotice.java
package com.summit.chat.model.entity;

import lombok.Data;

@Data
public class SysNotice {

  private Long id;
  private String msg;
  private java.sql.Timestamp createTime;
  private java.sql.Timestamp endTime;
  private Long version;
  private Long isDeleted;
  private Long publisherId;
  private Long like;
  private boolean isLike;


}

File: Chat-Web/src/main/resources/static/Chat/src/types/sysNotice.ts
/**
 * 系统公告类型定义
 */
export interface SysNotice {
    id?: number;
    msg: string;
    createTime?: string;
    endTime?: string;
    publisherId?: string;
    publisherName?: string;
}

export interface SysNoticeDto {
    id?: number;
    msg: string;
    endTime?: string;
    publisherId?: string;
}
